### ISSUE TITLE
Item 7 - Automating Systrace with MonkeyRunner fails

### ISSUE THREAD

**Issue by pgulotta** [googlecodelabs/android-perf-testing#14]
FYI:Running the test on Windows 10

Your command:   gradlew :app:assembleDebug :app:assembleDebugAndroidTest   ...
I think list an incorrect task, :app:assembleDebugAndroidTest.

Should be:  gradlew :app:assembleDebug :app:assembleAndroidTest  ....

**Comment by kremkrem** [comment]
No, your suggestion is wrong - tasks are generated in every possible flavor, including the existrence of assembleDebugAndroidTest , which compiles the test app destined to work with the debug app, compiled with assembleDebug. Meanwhile assembleAndroidTest I believe is supposed to compile the test app in all flavors, not just the one you really need.
